src/App.tsx

```diff
@@ -1,24 +1,194 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useContext,
+  useEffect,
+  useState,
+  useRef,
+} from 'react';
+import { AuthContext } from './components/Auth/AuthContext';
+
+import { postTodos, getTodos, deleteTodos } from './api/todos';
+import { ErrorType } from './types/ErrorType';
+import { Filter } from './types/Filter';
+
 import { UserWarning } from './UserWarning';
+import { TodoList } from './components/TodosList/TodosList';
+import { Header } from './components/Header/Header';
+import { Footer } from './components/Footer/Footer';
+import { Error } from './components/Error/Error';
 
-const USER_ID = 0;
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
+  const user = useContext(AuthContext);
+  const newTodoField = useRef<HTMLInputElement>(null);
+
+  const [isInputDisabled, setIsInputDisabled] = useState(false);
+  const [filterType, setFilterType] = useState(Filter.all);
+  const [isCreating, setIscreating] = useState(false);
+  const [title, setTitle] = useState('');
+  const [typeOfError, setTypeOfError] = useState(ErrorType.success);
+  const [processings, setProcessings] = useState<number[]>([]);
+  const [todos, setTodos] = useState<Todo[]>([]);
+
+  const loadingTodosApi = () => {
+    const handleRequest = (todoList: Todo[]) => {
+      setTodos(todoList);
+      setTypeOfError(ErrorType.success);
+    };
+
+    if (user) {
+      getTodos(user.id)
+        .then(list => handleRequest(list))
+        .catch(() => setTypeOfError(ErrorType.get));
+    }
+  };
+
+  useEffect(() => {
+    if (newTodoField.current) {
+      newTodoField.current.focus();
+    }
+
+    loadingTodosApi();
+  }, []);
```

Comment:
the user dependency should be added
```suggestion
  }, [user]);
```